lms/djangoapps/third_party_surveys/tasks.py

```diff
@@ -0,0 +1,58 @@
+from celery.schedules import crontab
+from celery.task import periodic_task
+from django.conf import settings
+from surveygizmo import SurveyGizmo
+from django.contrib.auth.models import User
+
+from lms.djangoapps.third_party_surveys.models import ThirdPartySurvey
+
+
+@periodic_task(run_every=crontab())
+def get_third_party_surveys():
+    """
+    Periodic Task that will run on daily basis and will sync the response data
+    of all surveys thorough Survey Gizmo APIs
+    """
+    survey_client = SurveyGizmo(
```

Comment:
Can we write a wrapper around survey gizmo? Its a good practice to not use such services directly in app